extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/Dialects.java

```diff
@@ -23,7 +23,7 @@ public static String guessDialect(String persistenceUnitName, String resolvedDbK
             return "io.quarkus.hibernate.orm.runtime.dialect.QuarkusH2Dialect";
         }
         if (DatabaseKind.isMariaDB(resolvedDbKind)) {
-            return "org.hibernate.dialect.MariaDB103Dialect";
+            return "org.hibernate.dialect.MariaDB106Dialect";
```

Comment:
Not entirely sure about this one. But to quote the comment further up:
> // For now select the latest dialect from the driver

If it's get's merged liek this we should probably add something to the migration guide? 🤔

Reply:
Agreed on the change and the note.

Reply:
:+1: thanks

Reply:
Done: https://github.com/quarkusio/quarkus/wiki/Migration-Guide-2.8#hibernate-mariadb-dialect-updated-to-106